content/ja/handbook/customer-success/pre-sales-post-sales-transition.md

```
@@ -0,0 +1,151 @@
+
+商談が商業的な議論を進める間、CSM は適宜 SA と連携し、組織内の戦略的な担当者や意思決定者との関係を維持する必要があります。
+
+### SAE と SA から CSM への責任の移行
+
+商談が[ステージ](/handbook/sales/field-operations/gtm-resources/) `Closed Won` に移行すると、CSM は顧客とのキックオフミーティングをスケジュールし、顧客の目標と CSM プログラムについてより詳細に話し合う必要があります。このミーティングの実施方法については、[CSM キックオフコールの詳細](/handbook/customer-success/csm/onboarding/#kickoff-call)をご確認ください。
+
+アカウントがポストセールスに移行した段階で、CSM はガイダンスとベストプラクティスの会話、顧客のイネーブルメント、製品使用に関する主要な責任を担います。SAE と SA の責任のほとんどは、この時点で CSM に移行します。
+
+顧客オンボーディング中、SA は顧客と引き続き関わり、シームレスな紹介を促進し、POV からの継続的な活動に対応することを選択できます。これは、CSM がそれらの活動を引き継げるように行われるべきであり、これらの項目に関する顧客とのすべての会話には CSM を含める必要があります。
+
+アカウントが完全にポストセールスに移行した後、SA は CSM から[特定の状況で再エンゲージするよう](#re-engaging-the-saeae-and-sa)招待される場合があります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)